scripts/deploy-enhanced.sh

```
@@ -169,6 +169,11 @@ zero_downtime_deploy() {
         ln -sfn deployments/${TIMESTAMP} current
         cd current
         source .env
+        
+        # Fix Redis system-level memory setting on server
+        echo '🔧 Applying Redis system-level optimizations...'
+        bash "$(dirname "$0")/fix-redis-sysctl.sh"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)